fh-android-sdk/src/main/java/com/feedhenry/sdk/api/FHAuthRequest.java

```diff
@@ -256,10 +258,15 @@ public void onReceive(Context pContext, Intent pIntent) {
                             resJson.put(FHAuthSession.SESSION_TOKEN_KEY, sessionToken);
                             resJson.put(
                                 "authResponse",
-                                new JSONObject(URLDecoder.decode(queryMap.get("authResponse"))));
+                                new JSONObject(
+                                    URLDecoder.decode(
+                                        queryMap.get("authResponse"),
+                                        Charset.defaultCharset().name())));
```

Comment:
Should we be using defaultCharset() here? We're making a call to a FeedHenry endpoint over the web so, in my view, it should be UTF-8 by default, rather than the client's default.